librz/type/path.c

```diff
@@ -289,10 +375,18 @@ RZ_API RZ_OWN RzList /*<RzTypePath *>*/ *rz_type_db_get_by_offset(const RzTypeDB
  */
 RZ_API ut64 rz_type_db_struct_member_packed_offset(RZ_NONNULL const RzTypeDB *typedb, RZ_NONNULL const char *name, RZ_NONNULL const char *member) {
 	rz_return_val_if_fail(typedb && name && member, 0);
+	eprintf("here\n");
 	RzBaseType *btype = rz_type_db_get_base_type(typedb, name);
+	eprintf("here\n");
 	if (!btype || btype->kind != RZ_BASE_TYPE_KIND_STRUCT) {
+		if (!btype) {
+			eprintf("return offset 0: !btype\n");
+		} else {
+			eprintf("return offset 0: btype->kind != RZ_BASE_TYPE_KIND_STRUCT\n");
+		}
 		return 0;
 	}
+	eprintf("here\n");
```

Comment:
just to remember to remove various debug printf.

Reply:
removed :)